lib/network/modules/PhysicsEngine.js

```diff
@@ -162,6 +162,18 @@ class PhysicsEngine {
           this.stopSimulation();
         }
 
+        if (this.options.enabled) {
+          const wind = this.options.wind;
+          if (wind) {
+            if (wind.x && typeof wind.x !== 'number' || isNaN(wind.x)) {
+              delete wind.x;
+            }
+            if (wind.y && typeof wind.y !== 'number' || isNaN(wind.y)) {
+              delete wind.y;
+            }
+          }
+        }
```

Comment:
I'm not saying this wouldn't work because it would but:
- `wind.x && ` doesn't do anything there.